### ISSUE TITLE
Excellent, but too expensive !

### ISSUE THREAD

**Issue by Overscore** [ccadic/sharpikeebo#4]
150$ + 18$ shipping to Europe + battery + Rpi Zero + case is definitely too expensive !  
A real shame!  
Quel domage !

**Comment by ccadic** [comment]
we understand this point of view. Europe design with european workers and european taxes + european social human protection standards + european work rules + european assembly + european process for certifications + random components prices at the moment due to shortage (obliging to take a security margin). The risk when launching a new device cannot be a suicide :) of course, once posisble we will review when possible.

**Comment by Overscore** [comment]
I understand that the company cannot sell at a loss, otherwise there is no more company...  But it's still a shame, I like this product so much.  
  
Je comprends que l'entreprise ne peut pas vendre à perte, sinon il n'y a plus d'entreprise… Mais c'est tout de même dommage, ce produit me plait tellement.

**Comment by ccadic** [comment]
yes sure. From the beginning , we are fighting to solve problems. This is another one (the rpi zero is also expesive now :) ) . so we keep fighting... :)
we are at the first stages of this product launch. This is made in the crowdfunding philosophy. Small qties = more expensive prices.  there is nothing we can do about this yet.  but our interest is , yes, to be able to drop the price in a range everyone is happy. with higher quantities, and less component shortage , the production price is obviously going to be better. 

oui bien sur. Depuis le début nous nous battons pour résoudre un a un les problemes. Ca n'est qu'un probleme de plus sur lequel de focaliser :)  le rpi zero est devenu tres cher aussi à cause de la penurie
nous somme cependant dans la phase de lancement et dans l'esprit de financement participatif par des early adopters. La petite serie et le choix du moins asiatique possible fait que les prix sont plus elevés.  bien evidemment si les quantites sont plus importantes alors mathématiquement les couts de fabrication baisseront

on y travaille.

**Comment by bialystok** [comment]
this is not expensive but this is a loot of money
